makefile.shade

```diff
@@ -0,0 +1,51 @@
+use namespace="System.IO"
+use namespace="System.IO.Compression"
+use namespace="System.Linq"
+
+use-standard-lifecycle
+k-standard-goals
+
+#repack-x86 target='compile' if='Directory.Exists("src")'
+    @{
+        var buildDir= Path.Combine(Directory.GetCurrentDirectory(), "artifacts", "build");
+        var projectName = "Microsoft.AspNetCore.Mvc.Razor.ViewCompilation";
+        var projectNupkg = Files
+            .Include(Path.Combine(buildDir, projectName + "*.nupkg"))
+            .Where(path => !path.EndsWith(".symbols.nupkg", StringComparison.OrdinalIgnoreCase))
+            .OrderByDescending(f => f)    // On local builds multiple nupkgs are generated.
+            .First();
+        
+        Log.Info("Repacking Nupkg: " + projectNupkg);
+        
+        var extractToDirectory = projectNupkg + "-temp";
+        ZipFile.ExtractToDirectory(projectNupkg, extractToDirectory);
+
+        var runtimesDirectory = Path.Combine(extractToDirectory, "runtimes");
+        var win7x86Directory = Path.Combine(runtimesDirectory, "win7-x86", "lib", "net451");
```

Comment:
@Eilon - we have two assemblies that have to be used in mutually exclusive contexts (x86 and x64 don't work together). Placing it in the `runtimes` root directory lets NuGet sort this out for us. We did something similar with RazorTooling